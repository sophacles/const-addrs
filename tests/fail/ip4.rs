use const_addrs::ip4;

fn main() {
    let _ = ip4!("192.168.11");
}
