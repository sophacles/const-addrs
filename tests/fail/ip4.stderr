error: invalid IPv4 address syntax
 --> tests/fail/ip4.rs:4:18
  |
4 |     let _ = ip4!("192.168.11");
  |                  ^^^^^^^^^^^^
